src/foundry/common/data/data.mjs/index.d.ts

```diff
@@ -1,6 +1,7 @@
 export { AmbientLightData } from './ambientLightData';
 export { ActiveEffectData } from './activeEffectData';
 export { ActorData } from './actorData';
+export { ChatMessageData } from './chatMessageData';
```

Comment:
`ChatSpeakerData` is missing in the exports.

Reply:
It is not actually exported by foundry, it's only used internally. So that's fine.